docs/src/pages/components/data-grid/style/style.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Data Grid - Styling
+components: DataGrid, XGrid
+---
+
+# Data Grid - Styling
+
+<p class="description">The grid css can be easily overwritten</p>
+
+The following grid leverages the CSS customization API to match the Ant Design specification.
+
+{{"demo": "pages/components/data-grid/style/AntDesignGrid.js", "defaultCodeOpen": false}}
```

Comment:
Why is this not under the components section? It's about the customization using the `components`. No?

Reply:
I did a page dedicated to styling and css. As it's mostly style I put it there. Should I put it in component?

Reply:
I think it's better to keep them separate as we could explain more how to use the theme...

Reply:
Ahhh, so it's about customization. How about we rename "Style" to "Customization"? IMHO, it's the same as the components props, but why have two different pages.

Reply:
I mean, the "Components" page is also about customization, but I believe the "Customization" terminology is often referenced as the CSS one. @mbrookes is my assumption accurate or would you argue for something else?